Classes/FlowQueryOperations/NeosUiDefaultNodesOperation.php

```diff
@@ -11,6 +11,8 @@
  * source code.
  */
 
+use Neos\ContentRepository\Domain\NodeType\NodeTypeConstraintFactory;
+use Neos\ContentRepository\Domain\Projection\Content\TraversableNodeInterface;
```

Comment:
In this file, we implement everything against the TraversableNodeInterface instead of NodeInterface; so that we use the forward-compatible API already.